Microsoft.Toolkit.Graph.Controls/Controls/GraphPresenter/GraphPresenter.cs

```diff
@@ -0,0 +1,113 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using Microsoft.Graph;
+using Microsoft.Toolkit.Uwp.Helpers;
+using Newtonsoft.Json.Linq;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Microsoft.Toolkit.Graph.Controls
+{
+    /// <summary>
+    /// Specialized <see cref="ContentPresenter"/> to fetch and display data from the Microsoft Graph.
+    /// </summary>
+    public class GraphPresenter : ContentPresenter
+    {
+        /// <summary>
+        /// Gets or sets a <see cref="IBaseRequestBuilder"/> to be used to make a request to the graph. The results will be automatically populated to the <see cref="ContentPresenter.Content"/> property. Use a <see cref="ContentPresenter.ContentTemplate"/> to change the presentation of the data.
+        /// </summary>
+        public IBaseRequestBuilder RequestBuilder
+        {
+            get { return (IBaseRequestBuilder)GetValue(RequestBuilderProperty); }
+            set { SetValue(RequestBuilderProperty, value); }
+        }
+
+        /// <summary>
+        /// Identifies the <see cref="RequestBuilder"/> dependency property.
+        /// </summary>
+        /// <returns>
+        /// The identifier for the <see cref="RequestBuilder"/> dependency property.
+        /// </returns>
+        public static readonly DependencyProperty RequestBuilderProperty =
+            DependencyProperty.Register(nameof(RequestBuilder), typeof(IBaseRequestBuilder), typeof(GraphPresenter), new PropertyMetadata(null));
+
+        /// <summary>
+        /// Gets or sets the <see cref="Type"/> of item returned by the <see cref="RequestBuilder"/>.
+        /// Set to the base item type and use the <see cref="IsCollection"/> property to indicate if a collection is expected back.
+        /// </summary>
+        public Type ResponseType { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the returned data from the <see cref="RequestBuilder"/> is a collection.
+        /// </summary>
+        public bool IsCollection { get; set; }
+
+        /// <summary>
+        /// Gets or sets list of <see cref="QueryOption"/> representing <see cref="Microsoft.Graph.QueryOption"/> values to pass into the request built by <see cref="RequestBuilder"/>.
+        /// </summary>
+        public List<QueryOption> QueryOptions { get; set; } = new List<QueryOption>();
+
+        /// <summary>
+        /// Gets or sets a string to indicate a sorting order for the <see cref="RequestBuilder"/>. This is a helper to add this specific request option to the <see cref="QueryOptions"/>.
+        /// </summary>
+        public string OrderBy { get; set; }
```

Comment:
Wouldn't it make sense to make all these properties as DPs, and react (do the query again with the updated parameters) when they change?

Reply:
For this and the RequestBuilder, I didn't expect them to change much yet. I think that's a separate PR to add that functionality in?

Reply:
Opened issue https://github.com/windows-toolkit/Graph-Controls/issues/56 to track this for later.